packages/fxa-settings/src/components/App/index.tsx

```diff
@@ -105,137 +145,135 @@ export const App = ({
     }
   }, [metricsEnabled, config.sentry, config.version, loading, isSignedIn]);
 
-  useEffect(() => {
-    if (Storage.isLocalStorageEnabled(window)) {
-      let uniqueUserId = JSON.parse(
-        localStorage.getItem('__fxa_storage.currentAccountUid') || '{}'
-      );
+  return (
+    <Router basepath="/">
+      <AuthAndSetUpRoutes path="/*" />
+      <SettingsRoutes path="/settings/*" {...{ showRecoveryKeyV2 }} />
+    </Router>
+  );
+};
 
-      const accounts: Array<{ email: string; uid: string }> = JSON.parse(
-        localStorage.getItem('__fxa_storage.accounts') || '{}'
-      );
+const SettingsRoutes = ({
+  showRecoveryKeyV2,
+}: {
+  showRecoveryKeyV2?: boolean;
+} & RouteComponentProps) => {
+  // TODO: FXA-8098
+  // const { loading, error } = useInitialSettingsState();
+  const settingsContext = initializeSettingsContext();
 
-      const currentUser = Object.values(accounts).find(
-        (x) => x.uid === uniqueUserId
-      );
+  return (
+    <SettingsContext.Provider value={settingsContext}>
+      <ScrollToTop default>
```

Comment:
This is the same as `<ScrollToTop default={true}>`. I was going to say whatever lib this is from probably accepts it as a prop, but looks like it's our file lol, and it looks like it does nothing. I'm guessing we want to revisit that component at some point, it has a note about reach-router which we're not going to be using shortly, and it also references `document` 🥴 We could probably use a comment on FXA-5071 and can look at it there since it's related, I'll make note, thanks!

Reply:
BTW, you'll see this convention else, search for `enabled`.